commit a42bbdcf538ddd5ad3242240402d959f884c0d5f
Author: Alex Deucher <alexander.deucher@amd.com>
Date:   Thu Nov 29 19:22:07 2018 -0500

    drm/amdgpu/powerplay: fix clock stretcher limits on polaris (v2)
    
    [ Upstream commit de4aaab5cc9770a8c4dc13d9bfb6a83b06bba57e ]
    
    Adjust limits for newer polaris variants.
    
    v2: fix polaris11 kicker (Jerry)
    
    Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>
    Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/drivers/gpu/drm/amd/powerplay/smumgr/polaris10_smumgr.c b/drivers/gpu/drm/amd/powerplay/smumgr/polaris10_smumgr.c
index 5b67f575cd34..45629f26dbc2 100644
--- a/drivers/gpu/drm/amd/powerplay/smumgr/polaris10_smumgr.c
+++ b/drivers/gpu/drm/amd/powerplay/smumgr/polaris10_smumgr.c
@@ -1528,8 +1528,21 @@ static int polaris10_populate_clock_stretcher_data_table(struct pp_hwmgr *hwmgr)
 	efuse = efuse >> 24;
 
 	if (hwmgr->chip_id == CHIP_POLARIS10) {
-		min = 1000;
-		max = 2300;
+		if (hwmgr->is_kicker) {
+			min = 1200;
+			max = 2500;
+		} else {
+			min = 1000;
+			max = 2300;
+		}
+	} else if (hwmgr->chip_id == CHIP_POLARIS11) {
+		if (hwmgr->is_kicker) {
+			min = 900;
+			max = 2100;
+		} else {
+			min = 1100;
+			max = 2100;
+		}
 	} else {
 		min = 1100;
 		max = 2100;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1098_a42bbdcf538ddd5ad3242240402d959f884c0d5f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1098_a42bbdcf538ddd5ad3242240402d959f884c0d5f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
